### PR TITLE
[AL-5109][AL-5110] Add Video and Raster Segmentation Annotation Kinds

### PR DESCRIPTION
**Pull request by whistler**
* Create new module for Video and DICOM annotation kinds
* Add `VideoMaskAnnotation` and `DICOMMaskAnnotation`
* Update `NDJsonConverter` to support new annotation kinds
* Update case conversion util to produce acronyms in uppercase i.e `URI` instead of `Uri`.

Note: This PR includes changes from #999, please review #999 first.

### CODE REVIEW

File: tests/data/serialization/ndjson/test_dicom.py
Comment (vbrodsky) [inline tests/data/serialization/ndjson/test_dicom.py:117]: I really prefer that we start comparing class types and actual attributes instead of serialize/deserialize stuff. I have discovered some tests passed while the actual object we deserialized into was the wrong type (I think our test fixture was wrong in that case).
Do we agree on this? Perhaps do not need to do for this PR if too much work

**Review by vbrodsky — COMMENTED** [review]
I can approve this once I have approved a dependent PR

File: tests/data/serialization/ndjson/test_dicom.py
Comment (whistler) [reply tests/data/serialization/ndjson/test_dicom.py:117]: I agree with you, comparing python objects instead of only using a json to deserialize then serialize catches more problems.

**Review by vbrodsky — APPROVED** [review]

**Pull request merged**
